<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWaveletCWTSetting</class>
 <widget class="QDialog" name="DialogWaveletCWTSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wavelet CWT</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>select series</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DA::DADataManagerComboBox" name="comboBoxDataMgr"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>data preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableViewPreview"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBoxPeakAnalysis">
      <property name="title">
       <string>Wavelet CWT</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="labelMethod">
         <property name="text">
          <string>Method</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelScales">
         <property name="text">
          <string>Scales</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxWavelet">
         <item>
          <property name="text">
           <string>cgau1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cgau2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cgau3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cgau4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cgau5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cgau6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cgau7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cgau8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cmor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fbsp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gaus1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gaus2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gaus3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gaus4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gaus5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gaus6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gaus7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gaus8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mexh</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>morl</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>shan</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxSamplingRate"/>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelSamplingPeriod">
         <property name="text">
          <string>Sampling period</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxMethod">
         <item>
          <property name="text">
           <string>conv</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fft</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelWavelet">
         <property name="text">
          <string>Wavelet</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelSamplingRate">
         <property name="text">
          <string>Sampling Rate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxSamplingPeriod">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelAxis">
         <property name="text">
          <string>Axis</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QSpinBox" name="spinBoxAxis">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="DA::DADataManagerComboBox" name="comboBoxScales"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DADataManagerComboBox</class>
   <extends>QComboBox</extends>
   <header>DADataManagerComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogWaveletCWTSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
